react-native-libraries.json

```diff
@@ -9823,5 +9823,18 @@
     "examples": ["https://github.com/Malaa-tech/react-native-simple-keypad/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/Alwinator/react-native-http-bridge-refurbished",
+    "npmPkg": "react-native-http-bridge-refurbished",
+    "examples": [
+      "https://github.com/Alwinator/react-native-http-bridge-refurbished#example"
+    ],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": false,
+    "windows": true,
+    "macos": true
```

Comment:
Due to compatibility issues in the past, to accept the out-of-three platforms support, we require an example app which runs using `react-native-macos` fork and/or `react-native-windows` library based on selected platform.